JLio.Core/Models/FunctionSupportedValue.cs

```diff
@@ -12,12 +12,12 @@ public FunctionSupportedValue(IFunction function)
 
         public IFunction Function { get; }
 
-        public JToken GetValue(JToken currentToken, JToken dataContext, IExecutionContext context)
+        public SelectedTokens GetValue(JToken currentToken, JToken dataContext, IExecutionContext context)
         {
-            var result = Function.Execute(currentToken, dataContext, options);
-            if (result.Data == null) return JValue.CreateNull();
-            if (result.Data.Type == JTokenType.String)
-                return result.Data.ToString().Trim(CoreConstants.StringIndicator);
+            var result = Function.Execute(currentToken, dataContext, context);
+            if (result.Success == false)
+                context.LogError(CoreConstants.FunctionExecution, "Execute of function failed");
```

Comment:
Do we know which function it is? so we can add it in the log?

Reply:
this is fixed